content/usecaseItems.md

```diff
@@ -1,5 +1,5 @@
 ---
 example_usecase_id: 0
-example_usecase_name: Example usecase
-example_usecase_description: This is an example usecase.
+example_usecase_name: Multi-year prediction of drought and heat stress to support decision making in the wheat sector
+example_usecase_description: Wheat sector is heavily influenced by recent alterations in the frequency and severity of extreme climate events. There is a growing need of climate information for effective planning and adaptive actions to the near-term climate variability and change.
```

Comment:
I think that now there are too much text on the first page of the use case. Can we summarize the description like: 
`The wheat sector is heavily influenced by extreme climate events. Therefore, climate information is needed to support decision-making in the wheat sector.`
Or perhaps we can replace this description with author's names and affiliation? something like `B. Solaraju-Murali, et al. Barcelona Supercomputing Center`

Reply:
Good idea! We use the author names and affiliations now to replace the description!